nise/yaml_generators/ocp_on_cloud/generator.py

```diff
@@ -0,0 +1,148 @@
+#
+# Copyright 2020 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+import logging
+import os
+
+import yaml
+from nise.yaml_generators.aws.generator import AWSGenerator
+from nise.yaml_generators.azure.generator import AzureGenerator
+from nise.yaml_generators.ocp.generator import OCPGenerator
+
+
+LOG = logging.getLogger(__name__)
+
+
+def _load_yaml_file(filename):
+    """Local data from yaml file."""
+    if not os.path.exists(filename):
+        raise FileNotFoundError(f'Cannot find file "{filename}"')
+    yamlfile = None
+    if filename:
+        try:
+            with open(filename, "r+") as yaml_file:
+                yamlfile = yaml.safe_load(yaml_file)
+        except TypeError:
+            yamlfile = yaml.safe_load(filename)
+    return yamlfile
```

Comment:
```suggestion
from nise.__main__ import _load_yaml_file
```

Let's DRY this up. (Also, we should probably rename it to `load_yaml_file`. The leading `_` shouldn't be there. There's no reason for it to be a private function.